src/components/VmActions/index.js

```diff
@@ -94,9 +95,34 @@ class VmActions extends React.Component {
 
     this.state = {
       removePreserveDisks: false,
+      isMobile: false,
     }
 
     this.getDefaultActions = this.getDefaultActions.bind(this)
+    this.updateScreenType = this.updateScreenType.bind(this)
+  }
+
+  componentDidUpdate () {
+    this.updateScreenType()
+  }
+
+  componentDidMount () {
+    window.addEventListener('resize', this.updateScreenType)
+    this.updateScreenType()
+  }
+
+  componentWillUnmount () {
+    window.removeEventListener('resize', this.updateScreenType)
+  }
+
+  updateScreenType () {
+    const state = { isMobile: false }
+    if (window.innerWidth <= 600) {
```

Comment:
I'm not sure that using this check all the time is a good idea. I think better will be using Bootstrap CSS helpers for hiding some parts for certain resolution: https://getbootstrap.com/docs/3.4/css/#responsive-utilities In that case you don't need to adding a lot of handlers and checks, you'll just hide needless component.